package.json

```diff
@@ -40,31 +40,30 @@
     "semantic-release": "semantic-release"
   },
   "dependencies": {
-    "@typescript-eslint/experimental-utils": "^4.21.0"
+    "@typescript-eslint/experimental-utils": "^4.24.0"
```

Comment:
This change makes this PR be released as a patch version. Not a problem, but it's not reflected in the PR title so I'm not sure if you are aware of it.

Reply:
We could have kept the `chore` imo, as we're not really using anything new from the package.
Even if we did, the code would still be working as it was.

Reply:
We need to let users know the deps have been updated. It could end up in a new bug, or just for letting them know about a new version is used under the hood for whatever reason they find it convenient.

Reply:
> we're not really using anything new from the package.

This is a reason to avoid bumping this package so often by the way. We are not using anything in particular for the latest version, but we are forcing users using this plugin to bump such sub-dependency in their project, which could make them end up with several `@typescript-eslint/experimental-utils` because they have different version ranges. Not something really problematic, but at least to be aware of.